examples/c++/multiply/CMakeLists.txt

```
@@ -14,6 +14,7 @@ set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --expt-relaxed-constexpr")
 set(CMAKE_CUDA_FLAGS_RELEASE "")
 set(CMAKE_CUDA_FLAGS_DEBUG "${CMAKE_CUDA_FLAGS_DEBUG} -g -G -O0")
 # change the path to your Icicle location
+include_directories("../../.." "/icicle")
```

This should be sufficient
```suggestion
include_directories("../../../icicle")
```

---

done